monitor/supply/metrics.go

```diff
@@ -0,0 +1,29 @@
+package supply
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	cChainSupply = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "monitor",
+		Subsystem: "supply",
+		Name:      "cchain_total_ether",
+		Help:      "Token supply on the consensus chain in ether",
+	})
+
+	eChainSupply = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
technically, we only call the `omni_evm` the "echain". We don't call l1 the echain. Suggest rename to `l1erc20Supply`